bindings/pulp/bindings/server.py

```diff
@@ -84,8 +84,8 @@ def __init__(self,
         self.verify_ssl = verify_ssl
         self.ca_path = ca_path
 
-    def DELETE(self, path, body=None, log_request_body=True):
-        return self._request('DELETE', path, body=body, log_request_body=log_request_body)
+    def DELETE(self, path, queries=(), body=None, log_request_body=True):
```

Comment:
If you put ``queries`` at the end of the function signature, then it will be a backward-compatible change.